src/clients/Device.js

```diff
@@ -811,15 +813,13 @@ class Device extends EventEmitter {
   };
 
   releaseOwnership = (flasher: Flasher) => {
-    logger.log('releasing flash ownership ', { deviceID: this.getDeviceID() });
+    logger.info({ deviceID: this.getDeviceID() }, 'releasing flash ownership ');
     if (this._owningFlasher === flasher) {
       this._owningFlasher = null;
     } else if (this._owningFlasher) {
       logger.error(
-        'cannot releaseOwnership, ',
-        flasher,
-        " isn't the current owner ",
-        { deviceID: this.getDeviceID() },
+        { deviceID: this.getDeviceID(), flasher },
+        'cannot releaseOwnership, isn\'t  current owner'
```

Comment:
I've tried a little bit, but is not easy. So please decide if trailing commas on parameters should be used. 
Here is a warning from ESLINT

* functions is set to "ignore" by default for consistency with the string option.
* functions should only be enabled when linting ECMAScript 2017 or higher.